lightning_pose/data/datasets.py

```diff
@@ -330,12 +332,191 @@ def __getitem__(self, idx: int) -> HeatmapLabeledExampleDict:
 
         """
         example_dict: BaseLabeledExampleDict = super().__getitem__(idx)
-        if len(self.imgaug_transform) == 1 and isinstance(
-            self.imgaug_transform[0], iaa.Resize
-        ):
+        if len(self.imgaug_transform) == 1 and isinstance(self.imgaug_transform[0], iaa.Resize):
             # we have a deterministic resizing augmentation; use precomputed heatmaps
             example_dict["heatmaps"] = self.label_heatmaps[idx]
         else:
             # we have a random augmentation; need to recompute heatmaps
-            example_dict["heatmaps"] = self.compute_heatmap(example_dict)
+            example_dict["heatmaps"] = self.compute_heatmap(example_dict)   
         return example_dict
+
+# class MultiviewHeatmapDataset(torch.utils.data.Dataset):
```

Comment:
i am using some of the methods from HeatmapDataset which are exactly the same as MultiviewHeatmapDataset so i tought it would be ok to inheret from heatmap, should i replicate the methods and inheret from torch.utils.data.Dataset?

Reply:
@farzad-ziaie did you have another question on this one? I saw a comment in the PR email that isn't showing up on github